kartothek/io/testing/read.py

```diff
@@ -148,24 +149,24 @@ def _perform_read_test(
             assert isinstance(res, MetaPartition)
         result = [mp.data for mp in result]
 
-        def sort_by(dct):
-            return dct["core"].P.iloc[0]
+        def sort_by(obj: Any) -> Any:
```

Comment:
Don't have any experience with type annotations, yet. Does `Any` offer any value opposed to _not_ annotating?

Reply:
fixed. Mypy was complaining that the different variants of `sort_by` have different signatures and I've thought that this is because of the types. Turns out, they default to `Any` and the issue was that the parameters are named differently.